### PR TITLE
Added the code for K largest elements in increasing order #1752

### PR DESCRIPTION
**Pull request by Mugdha-Hazra**

### CODE REVIEW

**Comment by Mugdha-Hazra** [comment]
Hello @rish-singhal sir, I have added the code for finding K largest element in ascending order, sir can you please review my code, I have also updated the readme file.

File: Code/C++/K_largest_element.cpp [hunk 1]
```diff
@@ -0,0 +1,98 @@
+//CPP Code For Finding K Largest Element In An Array In Ascending Order.
+/*
+             Approach
+             ========
+           
+            We can efficiently implement this using a Min-Heap.
+            
+            1. We will create a min heap.
+            2. We will traverse through all the elements of the array
```
Comment (rish-singhal) [inline Code/C++/K_largest_element.cpp:9]: Just update here that you are using a min heap and inverting the numbers, i.e. inserting -1*number

File: Code/C++/K_largest_element.cpp [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    {
+        cout << -minHeap.top() << ",";
+        minHeap.pop();
+    }
+    return 0;
+}
+
+/*
+           Time-Complexity: O(nlog(k))
+           where n => size of the array , k => K'th largest number
```
Comment (rish-singhal) [inline Code/C++/K_largest_element.cpp:55]: K is not the kth largest element, it is the number of largest elements to find.

File: Code/C++/K_largest_element.cpp [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        if (minHeap.size() > k)
+        {
+            minHeap.pop();
+        }
+    }
+
+    cout << "\nK largest array element in ascending order are : ";
+    while (!minHeap.empty())
+    {
+        cout << -minHeap.top() << ",";
```
Comment (rish-singhal) [inline Code/C++/K_largest_element.cpp:47]: Give a space after comma while printing that is:
", "

File: Array/readme.md [hunk 4]
```diff
@@ -26,6 +26,7 @@ To create an array, you need to known the length (or size) of the array in advan
 
  * Insertion_at_start ----> [C++](/Code/C++/insertion_at_start.cpp) 
  * Kadane's Algorithm ----> [C++](/Code/C++/kadane_algo.cpp) 
+ * K Largest Element  ----> [C++](/Code/C++/K_largest_element.cpp)
  * Job Sequencing Problem ----> [C++](/Code/C++/job_sequencing_problem.cpp)
```
Comment (rish-singhal) [inline Array/readme.md:29]: And do sort this list in lexicographical order

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Array/readme.md (hunk 4 above)
Comment (rish-singhal) [reply Array/readme.md:29]: @Mugdha-Hazra  This is left.

Comment (Mugdha-Hazra) [reply Array/readme.md:29]: done @rish-singhal sir.

Comment (rish-singhal) [reply Array/readme.md:29]: Find duplicate in array ~ should be at the top

**Review by rish-singhal — CHANGES_REQUESTED** [review]

**Review by Mugdha-Hazra — COMMENTED** [review]
done @rish-singhal sir.

File: Array/readme.md [hunk 5]
```diff
@@ -35,3 +36,4 @@ To create an array, you need to known the length (or size) of the array in advan
  * Inverse of an array ----> [C++](Code/C++/inverse_of_an_array.cpp) | [Python](/Code/Python/inverseArray.py)
```
Comment (rish-singhal) [inline Array/readme.md:36]: can you please update the whole list, as J comes before K and Find comes above so on...

**Review by rish-singhal — CHANGES_REQUESTED** [review]

File: Array/readme.md (hunk 5 above)
Comment (Mugdha-Hazra) [reply Array/readme.md]: I'm really sorry sir last time I just overlooked the whole list, now it's done'. @rish-singhal

**Comment by Mugdha-Hazra** [comment]
@rish-singhal  sir, I had made all the changes as per your direction can you please review it once.

**Comment by Mugdha-Hazra** [comment]
Thank you, @rish-singhal  sir.

**Pull request merged**